rpm/osg-pki-tools.spec

```diff
@@ -34,6 +34,11 @@ rm -f %{buildroot}%{python_sitelib}/*.egg-info
 
 
 %changelog
+* Tue Mar 26 2019 Jeny Teheran <jteheran@fnal.gov> - 3.2.0
+- Change -f/--hostfile to -F/--hostfile
+- Switch argparsing to ArgumentParser
+- Show message per certificate when request or retrieval fails
```

Comment:
I'd like to see a bit more complete list.  Changed -T to -t.  Increased the number of retries from 20 to 40 (200 seconds).  Removed the use of utils.charlimit_textwrap().  I'm not sure if there are other significant changes.